class Error(RuntimeError):
    pass


class AuthFailure(Error):
    pass
